### PR TITLE
Update tools and Dart VM pages

### PR DESCRIPTION
**Pull request by kwalrath**
Also a couple of style tweaks.

@filiph do you want to review?

### CODE REVIEW

File: _config.yml [hunk 1]
```diff
@@ -12,7 +12,7 @@ baseurl: "" # the subpath of your site, e.g. /blog/
 # for now...
 # dartlang: "https://www.dartlang.org"
 # dart4web: "https://webdev.dartlang.org"
-dartlang: "http://localhost:4000"
+dartlang: ""
```
Comment (filiph) [inline _config.yml]: Why this?

File: src/_assets/stylesheets/_overwrites.scss [hunk 2]
```diff
@@ -1 +1,16 @@
...
+}
+
+/* Make code font more noticeable (without being obnoxious). */
+code {
+	color: #008f83
+}
+
+/* Make code-font links look like links. Should also change the hover color. */
+a code {
+    color: #0000EE;
```
Comment (filiph) [inline src/_assets/stylesheets/_overwrites.scss]: Please use `$default-link` here, to stay semantic.

**Comment by filiph** [comment]
Two nitpicks. Otherwise, LGTM.

File: _config.yml (hunk 1 above)
Comment (kwalrath) [inline _config.yml]: If dartlang != "", then you get offsite images for all links from dart-vm pages to dartlang pages.

File: src/_assets/stylesheets/_overwrites.scss (hunk 2 above)
Comment (kwalrath) [inline src/_assets/stylesheets/_overwrites.scss]: I was being quick-and-dirty, figuring Tom would make it right. But it doesn't hurt to use the right thing. :)

File: _config.yml (hunk 1 above)
Comment (kwalrath) [inline _config.yml]: In fact... I should remove the dartlang comment up above, too.

**Comment by kwalrath** [comment]
Merging now. We can, of course, iterate.

**Pull request merged**